vcstool/clients/git.py

```diff
@@ -439,6 +468,27 @@ def import_(self, command):
 
                 checkout_version = command.version
 
+        if command.subpaths:
```

Comment:
```suggestion
        if command.subpaths:
            # Setting the paths sparse-checkout deals with.
```
Particularly because I see `if command.subpaths` block appears multiple times in this method, I think briefing the purpose per each chunk is helpful.